mixer/src/migrate/index.ts

```diff
@@ -0,0 +1,12 @@
+import client from "../db";
+
+// Currently we only have a few migration so I don't bring in extra dependencies to manage it
+// when we add a dozen more, switch to db-mate
+export const migrate = async () => {
+  await Promise.all([
+    client.query('alter table turing.blocks add column if not exists collator_id varchar(255)'),
+    client.query('alter table turing.blocks add column if not exists state_root varchar(255)'),
+    client.query('alter table turing.blocks add column if not exists extrinsics_root varchar(255)'),
+    client.query('create index concurrently if not exists blocks_collator on turing.blocks (collator_id);')
+  ]);
+}
```

Comment:
I’m curious, what happens if an error happens within migrate code. Would it stop the entire problem. In other words, does the program have any catch all exceptions code at root level?

This function doesn’t return anything. I am not sure if you need it, but `return Promise.all()`; could return the result of the 4 promises to the caller.

Reply:
okay, if you are adding a catch all at the root level, we might want a sentry hook there as well, similar to https://github.com/OAK-Foundation/webapp-parse-backend/blob/d6c0e97b3039654b2a6ffffb27da0341f111e000/src/index.js#L96

That way, Sentry can capture and keep all the issues and we can review them when we need to.